admin/write-news.php

```diff
@@ -0,0 +1,113 @@
+<?php
+    session_start();
```

Comment:
Codacy found an issue: [The use of function session_start() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)